solidity/script/avs/DeployAVS.s.sol

```diff
@@ -35,6 +36,8 @@ contract DeployAVS is Script {
     Quorum quorum;
     uint256 thresholdWeight = 6667;
 
+    address KILN_OPERATOR_ADDRESS = 0x1f8C8b1d78d01bCc42ebdd34Fae60181bD697662;
```

Comment:
is there an easy way to test this for more operators?

Reply:
There's a mapping in the AVSDirectory which I can query and put it in the json file but I felt like this was enough.